lgc/patch/PatchCopyShader.cpp

```diff
@@ -122,16 +123,8 @@ bool PatchCopyShader::runImpl(Module &module, PipelineShadersResult &pipelineSha
     //
     argTys = {int32Ty, int32Ty, int32Ty, int32Ty, int32Ty, int32Ty, int32Ty, int32Ty, int32Ty, int32Ty};
     argInReg = {true, true, true, true, true, true, true, true, true, false};
-    argNames = {"globalTable",
-                gfxIp.major <= 8 ? "streamOutTable" : "esGsLdsSize",
-                gfxIp.major <= 8 ? "esGsLdsSize" : "streamOutTable",
-                "streamOutInfo",
-                "streamOutWriteIndex",
-                "streamOutOffset0",
-                "streamOutOffset1",
-                "streamOutOffset2",
-                "streamOutOffset3",
-                "vertexOffset"};
+    argNames = {"globalTable",      "esGsLdsSize",      "streamOutTable",   "streamOutInfo",    "streamOutWriteIndex",
+                "streamOutOffset0", "streamOutOffset1", "streamOutOffset2", "streamOutOffset3", "vertexOffset"};
```

Comment:
Interesting, is this what clang-format came up with?

Reply:
Yes. The check-format will report error if they are lined in multiple rows as before. Should I turn off clang-format for it?